AutoCADAutomation/AutoCADAutomation/CollectionHelper.cpp

```diff
@@ -349,6 +352,42 @@ void CollectionHelper::CollectLiftDimensions(AcDbEntity* entity)
 	}
 }
 
+void CollectionHelper::CollectLiftDimensionsAfterReconnect(AcDbEntity* entity)
+{
+	if (wcscmp(entity->layer(), L"LIFT_INSERT_DIMENSIONS") == 0)
+	{
+		if ((wcscmp(entity->isA()->name(), L"AcDbDimension") == 0))
+			CollectLinesWithAnAngle(entity, vecHorDimLiftPlaceHolders, vecVerDimLiftPlaceHolders);
+		if (wcscmp(entity->isA()->name(), L"AcDbRotatedDimension"))
+		{
+			if (wcscmp(entity->isA()->name(), L"AcDbText") == 0)
+			{
+				AcDbText* text = (AcDbText*)entity;
+				if (text)
+				{
+					if (wcscmp(text->textString(), L"(LIFT INSERTS)"))
+						entity->erase();

```

Comment:
Why are we deleting the dimensions here ?

Reply:
Removed extra\unnecessary code